### PR TITLE
added 2015-11-19 meeting minutes

### PR DESCRIPTION
**Pull request by pmuellr**
could someone review and merge?

### CODE REVIEW

File: wg-meetings/2015-11-19.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+Andreas:
+
+Error handling facilities.  Catching errors in asyncWrap is difficult, as you
+lose some context because of having to catch uncaughtException yourself.
+Perhaps add "unhandled except for this async event", should be straight-forward.
+Sounds like there was some hope of domains handling this, but looking to avoid
+that magic.
+
+Patrick: Let's get this in the
+[list of Async Wrap issues](https://github.com/nodejs/tracing-wg/issues/29)
```
Comment (AndreasMadsen) [inline wg-meetings/2015-11-19.md]: Actually it is already there (see `onerror`) :)

**Comment by AndreasMadsen** [comment]
:+1:

**Comment by Qard** [comment]
LGTM.

**Comment by bnoordhuis** [comment]
I wasn't there but LGTM.  Sorry I missed it, it wasn't on my calendar for some reason.

**Pull request merged**